### ISSUE TITLE
Error(crash) with demo_sds on matlab.

### ISSUE THREAD

**Issue by hi-one-gg** [bharath272/sds_eccv2014#5]
Hi, i use cuda-6.5 ,nvidia-340, no-gpu and caffe installation, mkl etc...
when i run demo_sds on matlab crash is happend. this is command screen.

> ...
>    ( many line ..... )
> ....
> I0824 19:19:50.779052  8457 net.cpp:402] Collecting Learning Rate and Weight Decay.
> I0824 19:19:50.779070  8457 net.cpp:167] Network initialization done.
> I0824 19:19:50.779080  8457 net.cpp:168] Memory required for data: 158297088
> F0824 19:19:53.303887  8457 syncedmem.cpp:91] CPU-only Mode
> **\* Check failure stack trace: ***
> die

So i debug the demo_sds.m in matlab.
Crash is occured 20 line

> demo_sds.m(20 line) : imagelist_to_sds({imname}, imgdir, mcgdir, ovoutdir, sptextdir, regspimgdir, featdir, refinedoutdir, scorefile,10);

how can i solve this problem.. @bharath272
